bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Title.java

```diff
@@ -81,4 +81,24 @@ default boolean isLinkDisabled() {
         return false;
     }
 
+    /**
```

Comment:
This is one case where the newly introduced `Link` interface and the `LinkHandler` should help. Rather than defining these attributes inline for this component only, we should add them to the `Link#getHtmlAttributes`. We can then return a `Link` from the `Title` (something that's already due to happen in https://github.com/adobe/aem-core-wcm-components/pull/1419) and use these attributes from there.